integreat_cms/cms/fixtures/test_data.json

```diff
@@ -1863,7 +1863,7 @@
       "currently_in_translation": false,
       "version": 2,
       "minor_edit": false,
-      "last_updated": "2022-09-11T08:04:39.746Z",
+      "last_updated": "2019-08-12T08:04:49.746Z",
       "creator": 9,
```

Comment:
I don't really understand this change? Is it a necessary change?

Reply:
Ah, I should have mentioned to this point in PR description😅 
This is to keep the translation status of English version be "outdated" even after the change in translation calculation.

For a page content "Wissenswertes über Augusburg (Trivia about Augusburg)" we had the following test data.

- German ("Wissenswertes über Augusburg")
          Version 1: public, 2019-08-12T08:05:39.369Z
- English ("Trivia about Augusburg")
          Version 1: public, 2019-08-12T08:04:39.746Z
          Version 2: review, 2022-09-11T08:04:39.746Z

Before the change in translation status calculation, the english translation was judged as "outdated" since the english version 1 was used for translation status calculation and the english version 1 was created earlier than the german version 1.

Now with the change the latest major version is used for translation status calculation regardless of published or not. This means the english version 2 is now compared to the german version 1 in translation status calculation. To keep "Trivia about Augusburg" sill "outdated" after this change, the date of english version2 should be between the date of English version 1 and that of German version 1.

Reply:
Ah okay, that makes sense :)